prime-router/src/test/resources/fhirengine/translation/hl7/schema/schema-read-test-06/ORU_R01/patient.yml

```diff
@@ -0,0 +1,13 @@
+elements:
+  - name: patient-last-name
```

Comment:
I am a bit lost. How are the yml changes relevant to the sender transform?

Reply:
Good point, it's not directly tied to the sender transform, but I wanted to boost some code coverage in the area while I was there. There were some schema reader blocks that were related to the `extends` field, which weren't exercised by existing tests, so everything in this file is related to that; test is `test read from file with extends` in `ConfigSchemaReaderTests.kt`